recipes/xtr/all/conanfile.py

```diff
@@ -72,6 +95,13 @@ def build(self):
         env_build_vars["EXCEPTIONS"] = \
             str(int(bool(self.options.enable_exceptions)))
         env_build_vars["LTO"] = str(int(bool(self.options.enable_lto)))
+        if self.get_option("enable_io_uring") == False: # io_uring is enabled by default
+            env_build_vars["CXXFLAGS"] += " -DXTR_USE_IO_URING=0"
+        if self.get_option("enable_io_uring_sqpoll") == True:
+            env_build_vars["CXXFLAGS"] += " -DXTR_IO_URING_POLL=1"
+        capacity = self.get_option("sink_capacity_kb")
+        if capacity != "default" and capacity != None:
+            env_build_vars["CXXFLAGS"] += " -DXTR_SINK_CAPACITY={}".format(int(capacity) * 1024)
```

Comment:
```suggestion
        env_build_vars["CXXFLAGS"] += " -DXTR_USE_IO_URING={}".format("1" if self.options.get_safe("enable_io_uring") else "0")
        if self.options.get_safe("enable_io_uring_sqpoll"):
            env_build_vars["CXXFLAGS"] += " -DXTR_IO_URING_POLL=1"
        capacity = self.options.get_safe("sink_capacity_kb", None)
        if capacity is not None:
            env_build_vars["CXXFLAGS"] += " -DXTR_SINK_CAPACITY={}".format(int(capacity) * 1024)
```
Here, None is a the default value. Otherwise, is False.

Reply:
I've added this with some modifications, as always defining `XTR_USE_IO_URING` would prevent io_uring from being auto-detected as the default behaviour